react/features/filmstrip/components/native/TileView.js

```diff
@@ -292,7 +292,7 @@ function _mapStateToProps(state, ownProps) {
         _aspectRatio: responsiveUi.aspectRatio,
         _columns: columns,
         _disableSelfView: disableSelfView,
-        _height: responsiveUi.clientHeight - (ownProps.insets?.top || 0),
+        _height: responsiveUi.clientHeight - (ownProps.insets?.top || 0) - (ownProps.insets?.bottom || 0),
```

Comment:
We let the bottom "bleed out" on purpose, because there is no meaningful obstruction (like the notch) and the experience seems better since there is more video visible. Can you pl share a before / after picture so we can discuss?

Reply:
Will upload one

Reply:
It turned out there isn't a difference. This height is used only for the minHeight and we then expand it as much as we can. Basically no matter of this height we let the bottom "bleed out". 

Therefore I plan to remove the `insets.bottom` from here and also from the calculations. While playing with it I noticed a few other things that I may need to adjust for the calculations.